common/scripts/x86_64/RHEL_7/remote/installDb.sh

```diff
@@ -65,7 +65,9 @@ __update_sources() {
 __install_postgres() {
   echo "Installing Postgres"
   sudo rm /etc/yum.repos.d/redhat-rhui.repo
-  yum install -y https://download.postgresql.org/pub/repos/yum/9.5/redhat/rhel-7-x86_64/pgdg-redhat95-9.5-3.noarch.rpm
+  if [[ $(rpm -q pgdg-redhat95-9.5-3.noarch) != "pgdg-redhat95-9.5-3.noarch" ]]; then
```

Comment:
keep this in a variable. this is used 3 times in two lines. no point in repeating it